components/MainNavigation/StyledNavigation.js

```diff
@@ -1,16 +1,29 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 const StyledNav = styled.nav`
-  position: fixed;
-  bottom: 0.05rem;
-  left: 50%;
-  transform: translateX(-50%);
-  display: flex;
-  justify-content: space-evenly;
-  background: #525af2;
-  width: 368px;
-  margin: 0 auto;
-  padding: 0.1em 1em;
-  border-radius: 5px;
+  ${({ variant }) => {
+    if (variant === "main") {
+      return css`
+        position: fixed;
+        bottom: 0.05rem;
+        left: 50%;
+        transform: translateX(-50%);
+        display: flex;
+        justify-content: space-evenly;
+        background: #525af2;
+        width: 368px;
+        margin: 0 auto;
+        padding: 0.1em 1em;
+        border-radius: 5px;
+      `;
+    } else if (variant === "language") {
+      return css`
+        display: flex;
+        justify-content: flex-start;
+        gap: 1em;
+        background: whitesmoke;
+      `;
+    }
+  }}
```

Comment:
This pattern is ok for 2 variants, but a general approach is writing a function for each variant with the `&&` operator:
```js
${({variant}) => variant === 'main' && css`
   ... css here
`}
${({variant}) => variant === 'language' && css`
   ... css here
`}

Reply:
Thank you, I didn't know.